drivers/drgx.js

```diff
@@ -0,0 +1,23 @@
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat } = require('../lib/utils.js');
+
+module.exports = async () => {
+  const tickers = await request('https://exchange.drgx.io/cmc/public/returnTicker');
+  const pairs = Object.keys(tickers);
+
+  return pairs.map((pair) => {
+    const [base, quote] = pair.split('-');
+    const ticker = tickers[pair];
+
+    return new Ticker({
+      base,
+      quote,
+      quoteVolume: parseToFloat(ticker.quoteVolume24hr),
+      baseVolume: parseToFloat(ticker.volume24hr),
+      close: parseToFloat(ticker.last),
+      high: parseToFloat(ticker.high24hr),
+      low: parseToFloat(ticker.low24hr),
```

Comment:
@abdul I will also add ask and bid price because it's available in API :)

Reply:
@sajcics Thank you, I am going to update the pull request.

Reply:
@sajcics I have updated the code, can you please review it again?